Packs/CiscoFirepower/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Cisco Firepower
+- Maintenance and stability enhancements.
```

Comment:
We should try and avoid the "Maintenance and.." comment as much as possible and in this case, the only reason for RN is indeed the description change.

```suggestion
- Updated arguments descriptions.
```